frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyEssentialRequirements.yml

```diff
@@ -1,4 +1,5 @@
 name: Essential requirements
+short_name: Essential skills and experience
```

Comment:
We shouldn't need to make any changes to the legacy questions as we are not changing anything with the CSV.